Configurations/unix-Makefile.tmpl

```diff
@@ -170,10 +170,10 @@ HTMLSUFFIX=html
 
 
 CROSS_COMPILE= {- $config{cross_compile_prefix} -}
-CC= $(CROSS_COMPILE){- $target{cc} -}
+CC= {- $target{cc} -}
```

Comment:
Presumably reason is that you cross-compile by calling clang with no prefix. Instead you rely on -target option...

Anyway, I'd also say that this is inappropriate. Instead just don't set any CROSS_COMPILE variable when using clang.